requirements.txt

```diff
@@ -1,2 +1,3 @@
 flake8==3.5.0
 pylint==1.8.2
+prettytable==0.7.2
```

Comment:
uhm, `alpha`, last release in 2013... any alternative :sweat_drops:

Reply:
https://code.google.com/archive/p/prettytable/

Reply:
Good shout. Have now rolled my own implementation.